youtube_dl/extractor/vtm.py

```diff
@@ -34,33 +34,23 @@ class VTMIE(InfoExtractor):
         },
         {
             'url': 'http://vtm.be/video/volledige-afleveringen/id/257107153551000',
-            'info_dict': {
-                'id': '257107153551000',
-                'ext': 'mp4',
-                'title': 'Blind Getrouwd afl.6',
-                'description': 'md5:9ed26e8486ad0c1ddade4b066d83100f',
-            },
-            'skip_download': True,
+            'only_matching': True,
         },
```

Comment:
`only_matching` is only matched against `_VALID_URL`, such tests are for reference purposes. There is no sense in several similar tests that cover one scenario.